docs/reference/cli.md

```diff
@@ -20,8 +20,13 @@ To authenticate yourself, run the `auth` command in the terminal:
 logfire auth
 ```
 
+You will be prompted to select a [data region](./data-regions.md) (EU or US). To specify this
+via the cli use `logfire auth --region eu` or `logfire auth --region us`
```

Comment:
@Viicos is this the correct syntax?

Reply:
I set the `--region` argument on the `logfire` command (similarly to `--logfire-url`), as these two arguments are used by all the other subcommands. So the actual syntax will be:

`logfire --region eu auth`